src/plugins/record.ts

```diff
@@ -248,6 +270,7 @@ class RecordPlugin extends BasePlugin<RecordPluginEvents, RecordPluginOptions> {
 
   /** Destroy the plugin */
   public destroy() {
+    this.subscriptions.forEach((unsubscribe) => unsubscribe())
```

Comment:
This can be replaced by `super.destroy()` (the BasePlugin unsubscribes from all events).